client/Components/joinMatch.jsx

```diff
@@ -0,0 +1,50 @@
+'use strict';
+import { bindActionCreators } from 'redux';
+import React from 'react';
+import { render } from 'react-dom';
+import { connect } from 'react-redux';
+import { toJS } from 'immutable';
+import * as Actions from '../actions/actions.js';
+import request from 'superagent';
+
```

Comment:
i am slightly confided why this is not in the actions but I think that you know why this is :) just flagging this to check but assume this is noffle :)